core/src/test/java/kafka/test/junit/RaftClusterInvocationContext.java

```diff
@@ -134,13 +110,14 @@ public static class RaftClusterInstance implements ClusterInstance {
         private final ConcurrentLinkedQueue<Admin> admins = new ConcurrentLinkedQueue<>();
         private final boolean isCombined;
 
-        RaftClusterInstance(AtomicReference<KafkaClusterTestKit> clusterReference, AtomicReference<EmbeddedZookeeper> zkReference, ClusterConfig clusterConfig, boolean isCombined) {
-            this.clusterReference = clusterReference;
-            this.zkReference = zkReference;
-            this.clusterConfig = clusterConfig;
-            this.isCombined = isCombined;
+        RaftClusterInstance(RaftClusterInvocationContext context) {
+            this.clusterReference = context.clusterReference;
```

Comment:
We don't need `clusterReference` and `zkReference`,, right?

Reply:
Please review it